src/components/game/components/board/components/card/Card.js

```diff
@@ -16,11 +21,51 @@ const suitCharToSymbol = (suitChar) => {
   }
 };
 
+const suitNumtoRank = (suitRank) => {
+  switch (suitRank) {
+    case "02":
+      return <>2</>;
+    case "03":
+      return <>3</>;
+    case "04":
+      return <>4</>;
+    case "05":
+      return <>5</>;
+    case "06":
+      return <>6</>;
+    case "07":
+      return <>7</>;
+    case "08":
+      return <>8</>;
+    case "09":
+      return <>9</>;
+    case "10":
+      return <>10</>;
+    case "11":
+      return <>J</>;
+    case "12":
+      return <>Q</>;
+    case "13":
+      return <>K</>;
+    case "14":
+      return <>A</>;
+    default:
+      return null;
+  }
+};
```

Comment:
You can try to cast suitRank to number then return it directly in cases from to 2 to 10 and then use a switch case for the rest of the cases

Reply:
Done.